### PR TITLE
fix: unescape html entities in page title

### LINKED ISSUE #1372: `'` is replaced with `&#39;` in title for post
Example: https://elk.zone/m.webtoo.ls/@patak/109722747892738830

### PR DESCRIPTION
**Pull request by userquin**
closes #1372

### CODE REVIEW

**Comment by userquin** [comment]
IIRC, that's not true in the context we're using it, I need to add a check to unescapce these entities: ensure the title is enclosed by double quotes: this one `"<title>"` or this one  `XXXX in Elk: "<title>"` can be safely escaped

File: composables/setups.ts [hunk 1]
```diff
@@ -22,6 +22,11 @@ export function setupPageHeader() {
       if (buildInfo.env !== 'release')
         titleTemplate += ` (${buildInfo.env})`
       return titleTemplate
+        .replaceAll('&#34;', '"').replaceAll('&quot;', '"')
+        .replaceAll('&#38;', '&').replaceAll('&amp;', '&')
```
Comment (danielroe) [inline composables/setups.ts:26]: what about using `tiny-decode` here too?

Comment (userquin) [reply composables/setups.ts]: done

**Comment by userquin** [comment]
We've a small problem, what if the account name plus server has more than 60 chars? The pattern should be `<account> in <server>: "<message>"` (it depends on i18n, for example in spanish `<account> en <server>: "<message>"`) and so if `<account> in <server>:` has more than 60 chars long we lost the initial `"`...

**Review by danielroe — APPROVED** [review]

**Pull request merged**